### ISSUE TITLE
Ugur's Response

### ISSUE THREAD

**Issue by mr-devs** [CySoc2022/CySoc2022.github.io#12]
Check off boxes when Ugur answers the questions listed on [this document](https://docs.google.com/document/d/1nEQtiXPR9dVWLIbA48qdTuWciqdOjsHAHh1kFHf1M2U/edit?usp=sharing). Create a new issue for answers that require extra work.

- [x] Registration Link 
- [x] What are cyber social threats?
- [x] Best paper award
- [x] COVID language
- [x] EasyChair submission link
- [x] COVID spotlight

**Comment by mr-devs** [comment]
Ugurs response can be seen in the document linked above. 

You'll see that there are a couple of items that we'll need to wait on, so I will leave this issue open in the meantime.

**Comment by mr-devs** [comment]
I believe that I've opened issues for all of these outstanding items so I will close this now.
